test/tests/fallback.json

```diff
@@ -4,8 +4,7 @@
   "description": "Test cases for fallback behaviour.",
   "defaultTestProperties": {
     "bidiIsolation": "none",
-    "locale": "en-US",
-    "expErrors": true
```

Comment:
Was dropping this intentional? All of the messages in this file are expected to experience fallback and emit at least one error.